options/options.html

```diff
@@ -14,6 +14,7 @@
                     <label for="adapter">Client</label>
                     <select id="adapter" class="form-control">
                         <option value="deluge">Deluge</option>
+                        <option value="qbittorrent">qBittorrent</option>
```

Comment:
Should also be mentioned in the README :)

Reply:
Yep, and while touching that, added Deluge and fixed ruTurrent spelling ;-)